assets/sass/_buttons.scss

```diff
@@ -83,3 +83,8 @@
         @apply border-gray-700 text-gray-700 bg-transparent;
     }
 }
+
+.btn-scroll-top {
+    @apply text-white text-center cursor-pointer opacity-50 rounded-sm mt-8 w-8 h-4;
```

Comment:
Is the cursor-pointer needed? I only ask because it's an href now, so it should be a pointer automatically.

Reply:
good point. i'm guessing we will get it for free now.